src/main/main.cpp

```diff
@@ -51,23 +61,75 @@ boolean timeDiff(unsigned long start, int specifiedDelay)
 	return (millis() - start >= specifiedDelay);
 }
 
+/**
+ * Method to post to server
+ */
+void writeToServer()
+{
+	if (timeDiff(statusMillis, STATUS_UPDATE_DELAY))
```

Comment:
Should this not be timeDiff(statusMillis, WRITE_TO_SERVER_DELAY)) i can't see a variable for STATUS_UPDATE_DELAY

Reply:
Sorry there is a status_update_delay variable, but should this be status update delay and not write to server delay?